.travis.yml

```
@@ -79,16 +92,10 @@ before_install:
     fi
 
   - export XMLLINT_INDENT="	"
-  - composer require squizlabs/php_codesniffer:${PHPCS_BRANCH} wp-coding-standards/wpcs:${WPCS_BRANCH} --no-update --no-suggest --no-scripts
-  - |
-    if [[ "${TRAVIS_BUILD_STAGE_NAME^}" == "Sniff" ]]; then
-      composer install --dev --no-suggest
-      # The Composer PHPCS plugin takes care of the installed_paths.
-    else
-      if [[ ${TRAVIS_PHP_VERSION:0:3} > "7.1" ]]; then composer require phpunit/phpunit:^7.0 --no-update --no-suggest --no-scripts;fi
-      composer install --no-dev --no-suggest --no-scripts
-      composer install-standards
-    fi
+
+install:
+  - if [[ ${WPCS_BRANCH} == "dev-develop" ]]; then travis_retry composer config minimum-stability dev; fi
+  - travis_retry composer install --dev --no-suggest
```

```suggestion
  - travis_retry composer install --no-suggest
```

A `--dev` install is the default, so `--dev` can be removed.